PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -369,6 +372,12 @@ def miniAOD_customizeCommon(process):
     process.deepTauIDTask = cms.Task(process.deepTau2017v2p1, process.slimmedTaus)
     task.add(process.deepTauIDTask)
 
+    phase2_common.toModify(process.rerunDiscriminationByIsolationMVADBnewDMwLTPhase2raw, PATTauProducer=_noUpdatedTauName)
```

Comment:
based on https://github.com/cms-sw/cmssw/pull/30341#issuecomment-656327868, I propose:
```python
for _prod in ["rerunDiscriminationByIsolationMVADBnewDMwLTPhase2raw","rerunDiscriminationByIsolationMVADBnewDMwLTPhase2"]:
    if hasattr(process,_prod): phase2_common.toModify(getattr(process,prod), PATTauProducer=_noUpdatedTauName)
```
@makortel do you have any better ideas?

Reply:
That looks fine for a customize function. The approach with cff fragments is that all modules are defined for all eras, and the ones to be actually run are handled by customizing Tasks/Sequences.

Reply:
Is that really better than the original if-statement? Maybe I miss an advantage of having the modifier included, but at the moment this looks more confusing to me.

Reply:
@kpedro88 please confirm your preferred solution.

Reply:
Either is fine

Reply:
Ok, I switched back to the previous